### ISSUE TITLE
Internet Explore selection.restore / focus jumps on top of editor

### ISSUE THREAD

**Issue by manttih** [froala/wysiwyg-editor#762]
I created a new button, which opens a tailored material bank from which users can select uploaded files etc. I use selection.restore and events.focus here quite a lot as I open the material bank to jQuery UI dialog. It works perfectly on other browsers, but a quite annoying bug on IE seems to be:
- if I have a lot of text on editor, .focus jumps on the top of the editor (the selection is still there, but it's confusing).
Tested on Windows 7 / IE 11.

**Comment by stefanneculai** [comment]
If you could make a jsFiddle for it that would be very helpful.

**Comment by manttih** [comment]
I'll need to check this further myself, as it seems this has more to do with the JQUERY UI -dialog itself.. I'll return if I can find the cause for this problem.

**Comment by stefanneculai** [comment]
I'm closing this due to lack of information. We'll reopen it if you can find the cause of the problem as being the editor.

**Comment by manttih** [comment]
I examined this further, and I think I found the bug. See here:
http://www.synergiafoxy.com/froala_selectionrestore.php. 
If i select the text SELECT THIS AND CLICK CUSTOM BUTTON! , choose add link and press the custom button, IE jumps on top. I did this on minimum settings, so there should be no interfering code. So I think this is a bug in the editor.

It only does this when i restore selection on Froala popup (for example via linkInsertButtons), it doesn't jump if I have a custom button straight on the main toolbar.
